release/Magarena/scripts/Scrap_Trawler.groovy

```diff
@@ -10,7 +24,7 @@
             return new MagicEvent(
                 permanent,
                 new MagicTargetChoice(
-                    card(MagicType.Artifact).cmcLEQ(cmc).from(MagicTargetType.Graveyard),
```

Comment:
change cmcLEQ to cmcLT, this is clearer and easier to read. Will need to add cmcLT to MagicCardFilterImpl first.